docs/data-formats.md

```diff
@@ -332,6 +332,24 @@ This allows to add a custom attribute to a class, interface, delegate, struct or
 </linker>
 ```
 
+### DynamicallyAccessedMembers attribute in implicit this parameter
```

Comment:
I would omit this section - this is a very special case which pretty much nobody should need but us - it's really only valid on System.Type. Also in itself this is not related to the XML format - from the XML's point of view this is just an attribute on a method - the "this" semantics is special to the `DynamicallyAccessedMembersAttribute`.